README.md

```diff
@@ -0,0 +1,37 @@
+This is the main development environment for working on Mozilla payments.
+
+TODO: non-development environments are coming soon.
+
+You will need:
+* [Python](https://www.python.org/)
+* [Docker](https://docs.docker.com/)
+  * On Mac you can run this with
+    [VirtualBox](https://www.virtualbox.org/) and
+    [Boot2Docker](http://boot2docker.io/)
+* [docker-compose](https://docs.docker.com/compose/)
+
+Build your environment:
+* Check out the following repositories somewhere on your machine:
+  * [solitude](https://github.com/mozilla/solitude/)
+  * [payments-service](https://github.com/mozilla/payments-service/)
+* Run ``python link.py`` to connect your source.
+* Run ``docker-compose build`` to build the containers.
```

Comment:
If they are images, I think you need to call ``docker-compose pull`` to get them off docker hub, not build.